staging/src/k8s.io/kubectl/pkg/cmd/create/create_role.go

```diff
@@ -60,7 +60,7 @@ var (
 	specialVerbs = map[string][]schema.GroupResource{
 		"use": {
 			{
-				Group:    "extensions",
+				Group:    "policy",
```

Comment:
Right, but that assumes a default cluster, what if someone is using one with extensions/v1beta1 enabled, I'd expect this to add policy/podsecuritypolicies rather than change extensions/podsecuritypolicies. We need both to work correctly.

Reply:
Really thanks for your apply! I'm sorry i miss that, now i add both versions. :)